codice/alliance
# CAL-58 updated ddf version to 2.10.0-SNAPSHOT and updated branding fo…

#### What does this PR do?
update ddf version to 2.10.0-SNAPSHOT and fixes upstream changes to branding
#### Who is reviewing it?
@troymohl @bdeining @glenhein 
#### How should this be tested?
1. Build alliance
2. Launch app
3. Verify admin console has correct branding
4. Install alliance apps
5. Verify alliance branding on the search ui.
#### Any background context you want to provide?

#### What are the relevant tickets?
https://codice.atlassian.net/projects/CAL/issues/CAL-58?filter=allopenissues
#### Screenshots (if appropriate)
#### Checklist:
- [X ] Update / Add Unit Tests

…r upstream changes
